docs/configuration.md

```diff
@@ -30,6 +30,8 @@
   are both on the same file (count, default 0 (no backpressure))
   e.g. backpressure = 10 [writer = 100.log, slowest reader = 89.log, delta = 11]
 * **hostname** - hostname used in logging/messages (default gethostname())
+* **remove_checkpoint_on_stop** - removes the checkpoint entry when the plugin
```

Comment:
As per https://github.com/trink/hindsight/issues/24#issuecomment-215191565
> 1. I would clarify stopping as exiting NOT due to a Hindsight shutdown (on a shutdown the checkpoint should still be preserved)

This should include termination since it prevents a terminated plugin from blocking pruning.